### ISSUE TITLE
Is this workig?

### ISSUE THREAD

**Issue by david-vendel** [jimdc/gender-reveal#5]
Because it doesnt color anything to me, even though I have the script installed and running (on Greasemonkey on Firefox).
Did I miss something?

![screenshot 2018-04-16 12 58 44](https://user-images.githubusercontent.com/33971649/38805452-215ca662-4176-11e8-87e7-232fa6d8db1d.png)

**Comment by jimdc** [comment]
Works on my Firefox v59 running on Mac OS 10, although I do test more often with Chrome. When you open up the web console with cmd+shift+K on mac or ctrl+shift+K on windows, does it say:

"Gender reveal v0.9 loaded"
"assigned target language as French"

or are there any errors?

**Comment by david-vendel** [comment]
I tried couple times, and sometimes it throws error:
`Loading failed for the <script> with source “https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js”.`
After I refreshed the lesson page, it worked and highlights the words.
Then sometimes it works directly as it is supposed to work. No idea why sometimes it works and sometimes only after page refresh, but great addon anyway. I just think red color is bit too hard to read.
